packages/generator/gulpfiles/gulpfile-exchange-connector.ts

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
We decided to keep all the file names in lower camelCase. There was a PR before to rename all the existing files. Since these files are tasks I would suggest rename something like **apiDownloadTask**